### ISSUE TITLE
m3u8

### ISSUE THREAD

**Issue by masnatacion** [alxhotel/chromecast-api#45]
Hi

How can i cast m3u8 files?

thanks

**Comment by masnatacion** [comment]
i know what is the problem DefaultMediaReceiver.js

its needed for stream video :

`options.autoplay = true`

so i think we will add

`media.contentType.indexOf('vnd.apple.mpegurl') !== -1`

**Comment by ahmedwalid05** [comment]
I had a similar issue with m3u8 and the mime type. 
I fixed it with including the content mime type with url instead of just the url. 
Example:
```
let resource = {
url :"https://multiplatform-f.akamaihd.net/i/multi/will/bunny/big_buck_bunny_,640x360_400,640x360_700,640x360_1000,950x540_1500,.f4v.csmil/master.m3u8",
contentType: "video/H264"
}
  device.play(resource , function (err) {
    if (!err) console.log('Playing in your chromecast')
  })
```

**Comment by alxhotel** [comment]
Fixed in 6e9ea3609c74cbc1baaac2559e15ff1be1a7fa5d and released in `0.3.4`.

You can now use the URL of an m3u8 file.
